<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWidget</class>
 <widget class="QWidget" name="ChatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="lbUser">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>当前用户 :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbUsername">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>80</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>扛把子</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbCamera">
           <property name="text">
            <string>摄像头</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbVideo">
           <property name="text">
            <string>观看直播</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbSound">
           <property name="text">
            <string>音频</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbBarrage">
           <property name="text">
            <string>弹幕</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="lbVideo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>450</width>
           <height>250</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="tbShow">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QTextEdit" name="teInput">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSend">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labelnfo">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>80</height>
          </size>
         </property>
         <property name="text">
          <string>用户列表</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="lwUserList">
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="lbC">
             <property name="text">
              <string>积分 ：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbCount">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="lbB">
             <property name="text">
              <string>余额 ：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbBalance">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btnCharge">
           <property name="text">
            <string>充值</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnFlower">
           <property name="text">
            <string>鲜花</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="btnPlane">
           <property name="text">
            <string>飞机</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRocket">
           <property name="text">
            <string>火箭</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QRadioButton" name="rbtnShowChat">
           <property name="text">
            <string>纯净模式</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnHall">
           <property name="text">
            <string>返回大厅</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
